doc/news/changes/README.md

```diff
@@ -0,0 +1,40 @@
+This folder contains changelog entries
+======================================
+
+Changes between different releases in deal.II are documented in the file 
+"changes.h" that is automatically created from the files in the directories
+
+  * "major", 
+  * "minor" and 
+  * "incompatibilities".
+
+Each of these files contain a short description of a change, the authors' names
+and the date where the latter two a separated by "<br>" from the first one. 
+A typical file could look like this:
+
+    New: We can do fancy stuff now.
+    <br>
+    (John Doe, YYYY/MM/DD)
+
+and is named "YYYYMMDDJohnDoe". File names for multiple contributions from the 
```

Comment:
Yes, both `YYYYMMDD_this` and `YYYYMMDD_that` will also work. We use the file name only for sorting.